tests/accelerators/test_accelerator_connector.py

```diff
@@ -111,13 +112,13 @@ def on_fit_start(self, trainer, pl_module):
             raise SystemExit()
 
     model = BoringModel()
-    trainer = Trainer(fast_dev_run=True, accelerator="ddp", gpus=2, callbacks=[CB()])
+    with pytest.deprecated_call(match=r"accelerator='ddp'\)` has been deprecated in v1.5"):
+        trainer = Trainer(fast_dev_run=True, accelerator="ddp", gpus=2, callbacks=[CB()])
```

Comment:
Why not use strategy directly ?

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/pull/10554#discussion_r750342837